config/locales/de_DE.yml

```diff
@@ -1140,13 +1139,13 @@ de_DE:
   ticket_column_unit_price: "Stückpreis"
   ticket_column_total_price: "Gesamtpreis"
   menu_1_title: "Läden"
-  menu_1_url: "/shops"
+  menu_1_url: "/ shops"
   menu_2_title: "Karte"
   menu_2_url: "/map"
   menu_3_title: "Erzeuger"
-  menu_3_url: "/producers"
+  menu_3_url: "/ producers"
   menu_4_title: "Gruppen"
-  menu_4_url: "/groups"
+  menu_4_url: "/ groups"
```

Comment:
Who is dealing with :de: translations? It's been already many times that I see this change, which is going to break their menus.

Reply:
No sure. This one came from the machine translation. I fixed it now and left a comment. But it's difficult to explain this hack to translators.

Reply:
Machine translation? Are they using google translate from Transifex without checking? It'd great to know whats happening because looks like is going to happen again.

Reply:
Yes, Kirsten used machine translation to bootstrap the German locale. I think someone else is going through and reviewing it, but it's really difficult to tell in Transifex that this is a string that needs special attention.